dockerize.sh

```diff
@@ -3,17 +3,29 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 IMAGE_NAME="boresha/agritrader"
 DOCKER_HOME="/agritrader"
-ARG_NUM=2
 
 if [ "$1" == "init" ]; then
-	docker build -t $IMAGE_NAME $DIR
+	docker build -t "$IMAGE_NAME" "$DIR"
```

Comment:
Is this for if the variable includes spaces?

Reply:
That's one case, just general safety in bash scripts, learned it recently.